src/FSharp.Data.GraphQL.Shared/Output.fs

```diff
@@ -0,0 +1,5 @@
+namespace FSharp.Data.GraphQL.Shared
+
+open System.Collections.Generic
+
+type Output = IDictionary<string, obj>
```

Comment:
I would put `NameValueLookup` into this file

Reply:
Don't you think it's kind of confusing to name the file `Output.fs` and there's just one line of code defining `Output` as an alias for `Dictionary<string, obj>` while the rest of the file, which is not small by the way, is all just the definition of `NameValueLookup` 🙂 ?

Reply:
I think that is fine. As `NameValueLookup` is an `Output`

Reply:
Or we can go opposide way and move this interface there

Reply:
> As NameValueLookup is an Output

ok. I moved `NameValueLookup` to `Output.fs`.